### PR TITLE
chore: update dark image test

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: tests/shared/imageTest.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function imageTest(component: React.ReactElement) {
 
         const element = (
           <ConfigProvider theme={{ algorithm }}>
-            <App>
+            <App style={{ background: algorithm === theme.darkAlgorithm ? '#000' : '' }}>
```
Comment (li-jia-nan) [inline tests/shared/imageTest.tsx:43]: 是不是写反了，不应该是 `algorithm === theme.darkAlgorithm ? '' : '#000'` 这样吗？

Comment (afc163) [reply tests/shared/imageTest.tsx:43]: App 不应该默认有 darkAlgorithm 的背景么？

Comment (MadCcc) [reply tests/shared/imageTest.tsx:43]: > App 不应该默认有 darkAlgorithm 的背景么？

目前没有，App 不等同于 body，还是希望用户直接写到 body 上

Comment (MadCcc) [reply tests/shared/imageTest.tsx:43]: > 是不是写反了，不应该是 `algorithm === theme.darkAlgorithm ? '' : '#000'` 这样吗？

应该没反吧，`#000` 是黑色

**Pull request merged**